README.md

```diff
@@ -1 +1,207 @@
-# oci-kubernetes-monitoring
\ No newline at end of file
+# Monitoring Solution for Kubernetes
+
+## About
+
+This provides an end-to-end monitoring solution for Oracle Container Engine for Kubernetes (OKE) and other forms of Kubernetes Clusters, 
+using Logging Analytics, Monitoring other Oracle Cloud Infrastructure (OCI) Services.
+
+## Logs
+
+This solutions offers collection of various logs of a Kubernetes cluster, out of the box into OCI Logging Analytics and offer rich analytics on top of it. 
+Users may choose to customise the log collection by modifying the out of the box configuration that it provides.
+
+### Kubernetes System/Service Logs
+
+OKE or Kubernetes comes up with some built-in services where each one has different responsibilities and they run on one or more nodes in the cluster either as Deployments or DaemonSets. 
+
+The following service logs are configured to collect out of the box:
+- Kube Proxy
+- Kube Flannel
+- Kubelet
+- CoreDNS
+- CSI Node Driver
+- DNS Autoscaler
+- Cluster Autoscaler
+- Proxymux Client
+
+### Linux System Logs
+
+The following Linux system logs are configured to collect out of the box:
+- Syslog 
+- Secure logs
+- Cron logs
+- Mail logs
+- Audit logs
+- Ksplice Uptrack logs
+- Yum logs
+
+### Control Plane Logs
+
+The following are various Control Plane components in OKE/Kubernetes.
+- Kube API Server
+- Kube Scheduler
+- Kube Controller Manager
+- Cloud Controller Manager
+- etcd
+
+At present, control plane logs are not covered as part of out of the box collection, as these logs are not exposed to customers OKE. 
+The out of the box collection for these logs will be available soon for generic Kubernetes clusters and for OKE (when OKE make it available for end users).
+
+### Application Pod/Container Logs
+All the logs from application pods writing STDOUT/STDERR are typically available under /var/log/containers/. 
+Application which are having custom log handlers (say log4j or similar) may route their logs differently but in general would be available on the node (through a volume).
+
```

Comment:
should me add a mention that these are collected through a generic source and the user can do special handling as required.